test/common/bindata.go

```diff
@@ -14,7 +14,7 @@ See the License for the specific language governing permissions and
 limitations under the License.
 */
 
-// Code generated by go-bindata.
+// Code generated for package common by go-bindata DO NOT EDIT. (@generated)
```

Comment:
My understanding is that this file is a generated file. So, it is generated in previous `commit`. But it seems generate again here in this `commit`. Any information for sharing for this?